aws_adfs/login.py

```diff
@@ -225,7 +225,6 @@ def login(
     if stdout:
         _emit_json(aws_session_token)
     elif printenv:
-        _emit_summary(config, aws_session_duration)
```

Comment:
What about instead redirecting the summary to stderr?

Reply:
Pull request has been updated.